### ISSUE TITLE
Support for absolute layout and java.awt.Point

### ISSUE THREAD

**Issue by luca-domenichini** [jacek99/javabuilders#5]
I see that support for absolute layout is missing in the project.
In my case I would find it very useful in order to provide customers with a simple embedded swing editor in my application, oriented to only particular components (like SCADAs).

I think it would be enough to support a syntax for the Component#setLocation(Point p) or Component#setLocation(int x, int y) methods. All other stuffs are already working.. (more generally speaking, as you already support a special syntax for Dimension (like 800x600), a similar syntax should be fine for Point as well..).
In addition, Container#setLayout(null) must be called somewhere.. something like this:

``` yaml
JFrame(name=frame, title=frame.title, size=800x600, defaultCloseOperation=exitOnClose):
    - AbsoluteLayout()
    - JButton(name=button, text=button.acton, location=100;50)
```

(I don't know which char to use to separate x from y in location... ; should fit be maybe..)

what do you think about that?

**Comment by luca-domenichini** [comment]
Visit my fork at https://github.com/mimmoz81/javabuilders to get what i've done...

Also, I introduced the RectangleAsValueHandler class to handle type Rectangle (as of method Component#setBounds(Rectangle))

I was not able to pull a request to you.. I'm new to github so maybe I don't get all the mechanics..

**Comment by luca-domenichini** [comment]
Actually it does need for AbsoluteLayout tag to be placed at the end of the yml file.. i need to check more... I'll update when improved.

**Comment by luca-domenichini** [comment]
It seems that the layoutmanager must be always the last tag to be declared in yml. 

Is that true? If this is correct, my commit is already done and absolute layout is working properly..

**Comment by jacek99** [comment]
Yes, if I recall correctly. It's been many years since I coded it :-)

I think it is covered in the PDF explicitly if memory serves me right.

**Comment by luca-domenichini** [comment]
Ok, so if you have time to spend, take a look at my fork and tell me what you think..
many thanks
